app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="footer">
+  <div class="footer-links">
+    <a href="#"><i class="fab fa-github"></i></a>
+    <a href="#"><i class="fab fa-instagram"></i></a>
```

Comment:
Will we have these social media links? We shouldn't be adding things if they're not going to work in the end.